Fix/effects gateway

Voronapxl opened the pull request.

Sandrro committed:
Merge pull request #10 from LeonDeTur/dev

Dev

Sandrro committed:
added risk rating and placeholder for description in risk table

Sandrro committed:
changed set of attributes fro output of collect_texts router

Sandrro committed:
Merge pull request #11 from LeonDeTur/feat/descriptionable_rating

Feat/descriptionable rating

Sandrro committed:
feat: updated method of links generation

Sandrro committed:
fix: removed admin center and is_city from output of coverage method

Sandrro committed:
Merge pull request #12 from LeonDeTur/feat/descriptionable_rating

feat: updated method of links generation

Sandrro committed:
feat: updated risk calculation algorithm, added texts and indicators to output

Sandrro committed:
Merge pull request #13 from LeonDeTur/feat/descriptionable_rating

feat: updated risk calculation, added texts and indicators to output

Sandrro committed:
feat: added dynamic description for social risk

Sandrro committed:
refactor: slightly optimized description generation

Sandrro committed:
Merge pull request #14 from LeonDeTur/feat/descriptionable_rating

Feat/descriptionable rating

Sandrro committed:
feat: updated descriptions

Sandrro committed:
Merge pull request #15 from LeonDeTur/feat/descriptionable_rating

feat: updated descriptions

Sandrro committed:
feat: added new router for risk-to-values table

Sandrro committed:
Merge pull request #16 from LeonDeTur/feat/generate_risk_values_table

feat: added new router for risk-to-values table

Sandrro committed:
fix: now admin center collecting correctly

Sandrro committed:
Merge pull request #17 from LeonDeTur/fix/urbanapi_fateway

fix: now admin center collecting correctly

Sandrro committed:
fix: add drop of value support categories with missing data

Sandrro committed:
Merge pull request #18 from LeonDeTur/fix/risk_values

fix: add drop of value support categories with missing data

Sandrro committed:
feat: added router for collection of named objects

Sandrro committed:
Merge pull request #19 from LeonDeTur/feat/named_objects

feat: added router for collection of named objects

Sandrro committed:
feat:added list of services per value category

Sandrro committed:
Merge pull request #20 from LeonDeTur/feat/generate_risk_values_table

feat:added list of services per value category

Sandrro committed:
fix: removed excessive logger outputs

Sandrro committed:
Merge pull request #21 from LeonDeTur/feat/generate_risk_values_table

fix: removed excessive logger outputs

Sandrro committed:
fix: correction of collection from values_identites api

Sandrro committed:
Merge pull request #22 from LeonDeTur/feat/generate_risk_values_table

fix: correction of collection from values_identites api

Sandrro committed:
feat: added preprocessing logic and routers

Sandrro committed:
feat: finished base version of database methods

Sandrro committed:
refactor: fixed formatting with black

Sandrro committed:
fix: added parameter for address extraction

Sandrro committed:
refactor: deleted excessive imports

Sandrro committed:
added router for manual upload of messages, some fixes

Sandrro committed:
feat: added router and logic for named objects upload

Sandrro committed:
fix: updated reqs

Sandrro committed:
refactor: optimized functions

Sandrro committed:
refactor: separated logic for analysis to modules

Sandrro committed:
refactor: methods get data from soika db now

Sandrro committed:
refactor: cleanup, requirements changed

Sandrro committed:
Merge pull request #24 from LeonDeTur/refactor/calculation_update

Refactor/calculation update

Sandrro committed:
fix

Sandrro committed:
Merge pull request #25 from LeonDeTur/refactor/calculation_update

fix

Sandrro committed:
reverted changes

Sandrro committed:
reverted changes

Sandrro committed:
fixed reqs

Sandrro committed:
Merge pull request #26 from LeonDeTur/refactor/calculation_update

fixed reqs

Sandrro committed:
Revert "reverted changes"

This reverts commit 00df0ff5880a14375dc596ce9453f0eccc38c9b4.

Sandrro committed:
Revert "reverted changes"

This reverts commit 9c0e594c076c053c174b97988786acf7671bcf84.

Sandrro committed:
feat: changed get_texts_for_territory to data format for time series plot

Sandrro committed:
Merge pull request #27 from LeonDeTur/feat/time_series

feat: changed get_texts_for_territory to format for time series plot

Sandrro committed:
feat: mapped services to categories

Sandrro committed:
Merge pull request #28 from LeonDeTur/feat/time_series

feat: mapped services to categories

Sandrro committed:
feat: updated time series method

Sandrro committed:
Merge pull request #29 from LeonDeTur/feat/time_series

feat: updated time series method

Sandrro committed:
feat: added router for risk provision calculation

Sandrro committed:
Merge pull request #30 from LeonDeTur/feat/risk_values

feat: added router for risk provision calculation

Sandrro committed:
fix: fixed collection from db

Sandrro committed:
feat: added router and logic for risk-effects calculation

Sandrro committed:
Merge pull request #32 from LeonDeTur/feat/effects

feat: added router and logic for risk-effects calculation

Sandrro committed:
Merge branch 'master' of https://github.com/LeonDeTur/soika_api into feat/preprocessing

Sandrro committed:
refactor: controllers, logic, modules, db etc

Sandrro committed:
fix: post-refactor fixes

Sandrro committed:
fix: after-refactor fixes

Sandrro committed:
refactor and fixes: post-review edition

Sandrro committed:
refactor and fixes: post-review update

Sandrro committed:
Merge pull request #23 from LeonDeTur/feat/preprocessing

Feat/preprocessing

Sandrro committed:
fix: fixed response form for effects method

Sandrro committed:
Merge pull request #34 from LeonDeTur/fix/effects

fix: fixed response form for effects method

Sandrro committed:
feat: added social groups data for risk-provision and risk-effects routers

Sandrro committed:
Merge pull request #35 from LeonDeTur/feat/social_groups

feat: added social groups data for risk-provision and risk-effects

Voronapxl committed:
feat(llm_gateway): additional logic for GPU-serv calculations

Voronapxl committed:
fix(effects_gateway): effects API requests logic for when no effects data presented

LeonDeTur commented on app/common/api/llm_gateway.py:
```diff
@@ -0,0 +1,101 @@
+        endpoint = f"{self.url.rstrip('/')}/{extra_url.lstrip('/')}"
+        try:
+            async with self.session.get(endpoint, params=params) as response:
+                # если ответ не 2xx, это бросит ClientResponseError
```
убрать, лишний комментарий

LeonDeTur reviewed (changes requested):
rollback before gpu_access and rerequest without this commit

Voronapxl committed:
fix(effects_gateway): effects API requests logic for when no effects data presented (without GPU server commit)

# Conflicts:
#	app/common/api/effects_api_gateway.py
#	app/risk_calculation/social_risk_controller.py

Voronapxl committed:
rollback(llm_gateway): rollbackof GPU logic

Voronapxl committed:
rollback(llm_gateway): deleted problem import
